model/Licensing/Properties/additionId.md

```diff
@@ -0,0 +1,29 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# additionId
+
+## Summary
+
+Provides a short, unique identifier to refer to a LicenseAddition.
+
+## Description
+
+An additionId contains a human-readable, short-form identifier for a
```

Comment:
(non-blocking) this should probably be encoded as a [Format->pattern](https://github.com/spdx/spdx-3-model/blob/ac23ca73c879f37b2a1b17867db56cf306aa4097/model/Core/Classes/DateTime.md?plain=1#L24)